packages/transaction-controller/src/utils/gas-fees.test.ts

```diff
@@ -142,6 +145,21 @@ describe('gas-fees', () => {
         ).toBeUndefined();
       });
 
+      it('to request maxFeePerGas if advanced gas fees enabled', async () => {
```

Comment:
Should these tests be `to saved X` since this overrides the value in the request?

Reply:
Sorry, they all fixed now